styles/mdn/catppuccin.user.css

```
@@ -0,0 +1,238 @@
+/* ==UserStyle==
+@name           MDN Catppuccin
+@namespace      github.com/catppuccin/userstyles/styles/mdn
+@homepageURL    https://github.com/catppuccin/userstyles/tree/main/styles/mdn
+@version        0.1.0
+@description    Soothing pastel theme for MDN
+@author         Catppuccin
+@updateURL      https://github.com/catppuccin/userstyles/raw/main/styles/mdn/catppuccin.user.css
+@license        MIT
+
+@preprocessor   less
+@var select lightFlavor "Light Flavor" ["latte:Latte", "frappe:Frappe", "macchiato:Macchiato", "mocha:Mocha*"]
```

Not sure how this got changed from the template but the default light flavor needs to be Latte.
```suggestion
@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappe", "macchiato:Macchiato", "mocha:Mocha*"]
```